### ISSUE TITLE
End match earlier if a team already won most rounds

### ISSUE THREAD

**Issue by deiw0s** [packetflinger/openra2#47]
I don't know if there is a command for this, but if a team already won 5/9 rounds it makes no sense to keep the match until 9th round.

**Comment by joe0x04** [comment]
Interesting, is that how it works in the original RA2 mod? Should this be an option or just the way it is?

**Comment by deiw0s** [comment]
Yes, in original RA2 works that way. In the video you can see that after a team already won 5/9 rounds, the match countdown starts again. I believe is better to leave it as the original RA2.

https://github.com/user-attachments/assets/8b2e46bb-86d5-4264-a5d5-03df5c9e7757
